src/components/Form/ManageUsers.jsx

```diff
@@ -24,118 +22,169 @@ import FormSection from './FormSection';
  * @name ManageUsers
  */
 
+const WebIdEditButton = ({ lockStatus, setLockStatus }) => (
+  <IconButton
+    onClick={(e) => {
+      e.preventDefault();
+      setLockStatus(!lockStatus);
+    }}
+  >
+    {lockStatus ? <LockOpen /> : <Lock />}
+  </IconButton>
+);
+
+const formRowStyle = {
+  margin: '20px 0'
+};
+
+const textFieldStyle = {
+  margin: '8px'
+};
+
+const clearProcessing = (dispatch) => {
+  setTimeout(() => {
+    dispatch({ type: 'CLEAR_PROCESSING' });
+  }, 3000);
+};
+
+const sendUserToSolid = async (userObject, dispatch, session, state) => {
+  if (!userObject.username && !userObject.webId) {
+    runNotification(`Operation failed. Reason: No WebId provided`, 5, state, dispatch);
+    return;
+  }
+
+  dispatch({ type: 'SET_PROCESSING' });
+
+  runNotification(
+    `Adding user "${userObject.givenName} ${userObject.familyName}" to Solid...`,
+    5,
+    state,
+    dispatch
+  );
+  let listUsers = await addUserToPod(session, userObject);
+  listUsers = await getUserListActivity(session, listUsers);
+
+  runNotification(
+    `User "${userObject.givenName} ${userObject.familyName}" added to Solid`,
+    5,
+    state,
+    dispatch
+  );
+
+  clearProcessing(dispatch);
+
+  await listUsers;
+};
+
+const renderWebId = (username) => {
+  const template = ['https://', '.solidcommunity.net/profile/card#me'];
```

Comment:
Want to make this template structure more robust

Reply:
I think could try the format that was given as a default from Community Solid Server, so may we'll get urls with "https://<servername>.com/", "/profile/card#me".

Reply:
Before we merge this, I want to add a few more templates (like the one you mentioned). By default the system will select the one associated with the `SOLID_IDENTITY_PROVIDER` configured in the env. The user can select a different template from a dropdown menu (pending) if they want.

Ideally, this PR won't result in any actual UX changes for the people using the system. A case manager will still be adding first name, last name, username, then clicking `Add User`. The WebId field will only be there in case PASS can't figure out what the correct WebId should be.